--- gcc-9.2.0/contrib/download_prerequisites	2021-10-06 10:32:45.000000000 -0600
+++ gcc-9.2.0/contrib/download_prerequisites.new	2021-10-06 10:34:07.000000000 -0600
@@ -32,7 +32,7 @@
 mpc='mpc-1.0.3.tar.gz'
 isl='isl-0.18.tar.bz2'

-base_url='ftp://gcc.gnu.org/pub/gcc/infrastructure/'
+base_url='http://gcc.gnu.org/pub/gcc/infrastructure/'

 echo_archives() {
     echo "${gmp}"
@@ -58,7 +58,7 @@
 if type wget > /dev/null ; then
   fetch='wget'
 else
-  fetch='curl -LO -u anonymous:'
+  fetch='curl -L'
 fi
 chksum_extension='sha512'
 directory='.'
@@ -219,7 +219,7 @@
 do
     if [ ${force} -gt 0 ]; then rm -f "${directory}/${ar}"; fi
     [ -e "${directory}/${ar}" ]                                               \
-        || ${fetch} --no-verbose -O "${directory}/${ar}" "${base_url}${ar}"       \
+        || ${fetch} --no-verbose -o "${directory}/${ar}" "${base_url}${ar}"       \
         || die "Cannot download ${ar} from ${base_url}"
 done
 unset ar
